services/logs2notifications/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# build the binary
+ARG GO_VERSION
+ARG UPSTREAM_REPO
+ARG UPSTREAM_TAG
+FROM golang:${GO_VERSION:-1.16.4} AS builder
```

Comment:
we just standardized all the other go services to use `FROM golang:1.18-alpine3.16 AS builder` - can this be easy updated to go 1.18?

Reply:
Probably, will fix it up